internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -336,6 +336,8 @@ private ConfigKeys() {
    */
   public static final String DEFAULT_READ_STRATEGY = "default.read.strategy";
   public static final String DEFAULT_OFFLINE_PUSH_STRATEGY = "default.offline.push.strategy";
+  public static final String DEFAULT_CONCURRENT_PUSH_DETECTION_STRATEGY = "default.concurrent.push.detection.strategy";
```

Comment:
Can we rename `DEFAULT_CONCURRENT_PUSH_DETECTION_STRATEGY` to `CONCURRENT_PUSH_DETECTION_STRATEGY`?
The current name suggests it’s the default value, but it appears to be a config key, not the default itself.